code/modules/busy_space_vr/organizations.dm

```diff
@@ -416,6 +416,45 @@
 	ship_prefixes = list("TTV" = "transport", "TTV" = "luxury transit")
 	destination_names = list()
 
+/datum/lore/organization/tsc/aphrodite //A tribute to another SS13 server that is no longer running. Many of the players migrated to yawn.
+	name = "Aphrodite Fulfillment Ltd."
+	short_name = "Aphrodite"
+	acronym = "ARF"
+	desc = "One of the less powerful TSCs, AphRodite Fulfillment Limited (ARF) focuses on consumer enjoyment and pleasure. \
+	While they cover a broad range of services, they are most known for their manufacture of exotic and erotic adult toys, as well as \
+	the interstellar 'pleasure cruises' that they offer which are frquently, though not always, adult in nature. They also offer other luxury \
+	services and products, such as custom-made prosthesises, unconventional (but not experimental) gene therapy, and luxury adult entertainment."				// One or two paragraph description of the organization, but only current stuff.  Currently unused.
+	history = ""
+	work = "luxury and adult goods and services"
+	headquarters = ""
+	motto = "Your pleasure is our pleasure"
+	ship_prefixes = list("ARFS" = "general supply", "ARFS" = "pleasure cruiser", "ARFS" = "transport")
+	ship_names = list(	//Generally love themed.
+		"Dallus", //the original ship map used by the ARFS server. Spelled with a u intentionally, because reasons.
+		"Endeavor", //another ship map used by the ARFS server. Citadel Station's RP server will likely use this map in future.
+		"Red Rocket",
+		"Silver Soul",
+		"Lusty Xenomorph Maid",
```

Comment:
no, please just no

Reply:
Basically, I'm not going to trade a snowflake company for another :v

Reply:
I figured that "Lusty Xenomorph Maid" would be in line with some of the other memey ship names from other TSCs (Morpheus' is a very big example of these) but I can absolutely remove it

Reply:
As for the snowflake company comment- This company is nowhere near as wide-reaching and snowflakey as KHI was. They're not "post scarcity", they don't have all the tech in the world. They're influence on the station as a whole is extremely limited if it exists at all, as opposed to KHI's "we made all of your fancy tech" stuff. The only place they will exist is in ATC chatter and Codex entries- as well as @DameonOwen's fluff cybernetics.

Reply:
Not just the ship names its the whole  "manufacture of exotic and erotic adult toys" and 'pleasure cruises'

Don't get me wrong there probably are companies that do that kind of stuff, but a Trans-Stellar Corporation, really don't want to add the BIG SEX COMPANY to our server's official lore.

"Company is nowhere near as wide-reaching and snowflakey as KHI" Well that is why it got removed and adding them here would not just place them in ATC chatter it would place them in official lore. Though we don't limit what people can come up on, on their own and use as their own characters lore, placing it in official lore is another thing.